lib/Qubit.class.php

```diff
@@ -351,4 +346,24 @@ public static function moveUploadFile($file)
     $file['tmp_name'] = $tmpFilePath;
     return $file;
   }
+
+  /**
+   * Create "uploads/" and "uploads/tmp/" if they don't exist
+   */
+  public static function createUploadDirsIfNeeded()
+  {
+    $uploadsPath = sfConfig::get('sf_upload_dir');
+    $directories = array(
+      $uploadsPath,
+      $uploadsPath . DIRECTORY_SEPARATOR . 'tmp'
+    );
+
+    foreach ($directories as $dirPath)
```

Comment:
Could use mkdir(..., ..., true) instead of a loop here to create both folders I think.

Reply:
Yes, but someone was complaining about some issues with the permissions on Windows, so I decided to leave it like that, at least we're not calling chmod(), heheh

Reply:
With this implementation if the need arose, we could add additional folders to the array.  🤔